utils/httpRequest/campaign.go

```diff
@@ -25,17 +25,17 @@ func HTTPEditCampaign(id, data string) ([]byte, error) {
 	return HTTPRequest(http.MethodPatch, utils.GetHost()+"/v1/accounts/"+viper.GetString("account_id")+"/account_environments/"+viper.GetString("account_environment_id")+"/campaigns/"+id, []byte(data))
```

Comment:
I think using constants instead of plain texts might help in case of future update of these texts